cmd/limits_test.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import "testing"
+
+func TestResourcesLimitsNil(t *testing.T) {
+	runTest(t, "resources_limit_nil.yml", auditLimits, ErrorResourcesLimitsNIL, "", "")
```

Comment:
`runTest` has optional parameters so the last arguments `, "", ""` are not needed.